lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -446,7 +445,12 @@ namespace Js
             // Clear disable implicit call bit as initialization code doesn't have any side effect
             Js::ImplicitCallFlags saveImplicitCallFlags = scriptContext->GetThreadContext()->GetImplicitCallFlags();
             scriptContext->GetThreadContext()->ClearDisableImplicitFlags();
-            JavascriptFunction::CallRootFunctionInScript(function, Js::Arguments(callInfo, args));
+
+            {
+                AutoSuspendDebuggerRecording autoSuspendDebuggerRecording(scriptContext);
+                JavascriptFunction::CallRootFunctionInScript(function, Js::Arguments(callInfo, args));
```

Comment:
This is the lazy-initialization of the Intl object. It cannot call user code, and everything about it is library-code-internal and unobservable by design.